ui/pages/confirmations/hooks/alerts/transactions/useFirstTimeInteractionAlert.test.ts

```diff
@@ -61,6 +73,9 @@ function runHook({
 describe('useFirstTimeInteractionAlert', () => {
   beforeEach(() => {
     jest.resetAllMocks();
+    (useTrustSignal as jest.Mock).mockReturnValue({
+      state: TrustSignalDisplayState.Unknown,
+    });
   });
```

Comment:
```suggestion
  const mockUseTrustSignal = jest.mocked(useTrustSignal);
  beforeEach(() => {
    jest.resetAllMocks();
    mockUseTrustSignal.mockReturnValue({
      state: TrustSignalDisplayState.Unknown,
    });
  });
```

Minor, this way we could have avoid unnecessary `(useTrustSignal as jest.Mock)` type conversions repetations.